### PR TITLE
initial commit with actual code

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: base.sh [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+localdir=$%(CSUB_SCRATCH_NODE)s/$jobname
+
+myecho () {
+    echo "$1"
+}
+
+if [ -z "$%(CSUB_SCRATCH_NODE)s" ]
+then
+    echo "%(CSUB_SCRATCH_NODE)s undefined"
+    env|sort|grep %(CSUB_ORG)s
```
Comment (stdweird) [inline base.sh]: :+1:

File: base.sh [hunk 2]
```diff
@@ -0,0 +1,549 @@
...
+    	then
+    		# copy initial job directory for array jobs
+    		cp -r $localdir_initial $localdir
+    	else
+        	## problem
+        	myecho "No localdir $localdir found (No shared checkpoint)"
+        	exit 1
+        fi
+    fi
+else
```
Comment (stdweird) [inline base.sh]: i sort of second that. if you have a block
```
if not condition
then 
   do_no
else
   do_yes
fi
```
you should always reorder it to
```
if condition
then
  do_yes
else
  do_no
fi
```
it's just more readable.

File: base.sh [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+jobname=${%(CSUB_JOBNAME)s}
+## jobname_stripped is set in BASEHEADER
+scriptname="${jobname_stripped}.sh"
+localdir=$%(CSUB_SCRATCH_NODE)s/$jobname
+
+myecho () {
+    echo "$1"
+}
+
+if [ -z "$%(CSUB_SCRATCH_NODE)s" ]
```
Comment (stdweird) [inline base.sh]: another style remark: avoid the newlines with only `then` or `do`; use
```bash
if [ -z  something ]; then
  ....
fii
...
for x in a b c; do
  ...
done
```

**Comment by boegel** [comment]
We should get this merged as is, and clean it up gradually, otherwise we'll end up rewriting the whole thing in the first (code) commit...

**Pull request closed without merging**